templates/app/views/home/_collections_with_call_to_action.html.erb

```diff
@@ -1,6 +1,6 @@
 <section class="grid-container grid-cols-1 gap-7 wrapper py-4 lg:py-12 lg:grid-cols-3">
   <div class="flex flex-col items-center justify-center py-10 px-3">
-    <h3 class="text-h2.5 text-center font-serif mb-5">Become the brand everyone talks about.</h3>
```

Comment:
Okay, this one seems like something that probably shouldn't be treated as a H3 element.